src/views/primary-auth/CustomButtons.js

```diff
@@ -27,16 +27,15 @@ define([
 
   return Okta.View.extend({
 
-    className: 'social-auth',
+    className: 'primary-auth-container',
 
     children: function () {
       var children = [],
-          idProviders = this.settings.get('configuredSocialIdps'),
-          divider = dividerTpl({text: Okta.loc('socialauth.divider.text', 'login')});
+        buttons = this.settings.get('configuredButtons'),
+        divider = dividerTpl({text: Okta.loc('authbutton.divider.text', 'login')});
```

Comment:
Hmm, I am a little hesitant about this change. It is a breaking change - should think about this more (i.e. if someone updates the widget, is customizing this string, and is using socialAuth, this will result in a change of behavior for them).